src/CONST.ts

```diff
@@ -353,6 +353,7 @@ const CONST = {
         OLD_DOT_ANDROID: 'https://play.google.com/store/apps/details?id=org.me.mobiexpensifyg&hl=en_US&pli=1',
         OLD_DOT_IOS: 'https://apps.apple.com/us/app/expensify-expense-tracker/id471713959',
     },
+    URL_DEFAULT_SCHEME: 'http',
```

Comment:
I think it will be more clean if we rename it to `COMPANY_WEBSITE_DEFAULT_SCHEME` because it is the default for company website only, not all URLs.

Reply:
@Krishna2323 bump!